flow/layers/physical_shape_layer.cc

```diff
@@ -9,8 +9,15 @@
 
 namespace flutter {
 
+// These numbers have been chosen empirically to give a result closest to the
+// material spec.
+// Sync with /lib/web_ui/lib/src/engine/canvaskit/util.dart
+const SkScalar kAmbientAlpha = 0.039f;
+const SkScalar kSpotAlpha = 0.25f;
+const SkScalar kLightRadius = 1.1f;
 const SkScalar kLightHeight = 600;
-const SkScalar kLightRadius = 800;
+const SkScalar kLightXOffset = 0;
+const SkScalar kLightYOffset = -450;
```

Comment:
The code used to use an offset of -600 from the top of the bounds, but the value is now hard-coded as a -450 offset directionally. Doesn't that produce a different effect?

Reply:
Or is this change intentional due to undesired results of the previous fixed offset?

Reply:
I don't know. I just copied these values from [canvaskit/util.dart](https://github.com/flutter/engine/blob/master/lib/web_ui/lib/src/engine/canvaskit/util.dart). We should ask this to @yjbanov.

Reply:
The values we use on the web side don't have much science to them. We just eyeballed the result and checked with the material team. In the 6 months since the change we've received zero complaints about how the shadows look.